integration_tests/testsuite/driver.py

```diff
@@ -81,18 +84,16 @@ def _main():
             sys.exit(1)
 
         logging.debug('Deploying app!')
-        deploy_url = deploy_app(args.image, args.directory)
+        version, deploy_url = deploy_app.deploy_app(args.image, args.directory)
```

Comment:
One minor nit: If you make deploy_app just return the version, then you can move the call to retrieve_url to 89 and only call that function if an application_url wasn't passed in.